src/devices/interlogix.c

```diff
@@ -186,6 +187,14 @@ static int interlogix_decode(r_device *decoder, bitbuffer_t *bitbuffer)
         f3_latch_state = ((message[3] & 0xe) == 0xc) ? "CLOSED" : "OPEN";
         f4_latch_state = ((message[3] & 0xe) == 0x2) ? "CLOSED" : "OPEN";
         f5_latch_state = ((message[3] & 0xe) == 0xa) ? "CLOSED" : "OPEN";
+    // PET Immune SAW PIR motion sensor logic.
+    } else if ((reverse8(message[2]) >> 4) == 0x4) {
+        low_battery    = (message[3] & 0x10) ? "LOW" : "OK";
```

Comment:
Use `low_battery    = (message[3] & 0x10) ? 1 : 0;` (same as in the block below)

Reply:
Change made. Thanks for the review.